lib/compat/wordpress-6.7/class-gutenberg-rest-posts-controller-6-7.php

```diff
@@ -0,0 +1,699 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Posts_Controller_6_7 class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.7.0
+ */
+
+/**
+ * Extend the core WP_REST_Posts_Controller class to add support for post formats.
+ *
+ * @see WP_REST_Controller
+ */
+class Gutenberg_REST_Posts_Controller_6_7 extends WP_REST_Posts_Controller {
+	/**
+	 * Post type.
+	 *
+	 * @since 4.7.0
+	 * @var string
+	 */
+	protected $post_type;
```

Comment:
this var already appears to be part of `WP_REST_Posts_Controller` - why is being added in this subclass?

Reply:
That was my addition to satisfy the IDE PHP linter. Happy to revert if it's confusing. Ref: 8eab071a5ae68e6ea7fb43bf13834b10282db7b7.

Reply:
ah, your IDE wasn't aware of the base class? I guess it's not _wrong_ to redeclare it here. I'm not familiar enough to know if there are concerns about doing so; I just assumed there might be if we aren't changing its visibilty.